.eslintrc

```diff
@@ -12,6 +12,7 @@
 		"react-hooks"
 	],
 	"rules": {
+		"jsdoc/require-param": "off",
```

Comment:
This is already TypeScript, so JSDoc params in addition to that probably aren't needed.